File: Iniciante/1002.java
package uri;

import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

/*
 * 
*/
public class Main {
	public static void main(String[] args) throws IOException {
		Scanner sc = new Scanner(System.in);
		Locale.setDefault(Locale.US);
		double raio = sc.nextDouble();
		double PI = 3.14159;
		double A = Math.pow(raio, 2)*PI;
		System.out.println("A="+String.format("%.3f",A));
		sc.close();
	}

}

File: Beginner/1002.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var r float64
	const pi float64 = 3.14159

	fmt.Scanf("%f", &r)
	fmt.Printf("A=%0.4f\n", math.Pow(r, 2)*pi)
}
